pandas/core/frame.py

```diff
@@ -7292,8 +7292,9 @@ def corr(self, method="pearson", min_periods=1) -> "DataFrame":
 
         See Also
         --------
-        DataFrame.corrwith
-        Series.corr
+        DataFrame.corrwith : Compute pairwise correlation between rows or
+            columns of two DataFrame objects.
```

Comment:
remove

Reply:
just L7296 if accept above suggestion.

Reply:
Thanks for you feedback. I just incorporated your suggestions.